curtsies/escseqparse.py

```diff
@@ -65,69 +87,103 @@ def peel_off_esc_code(s):
     >>> d == {'numbers': [2], 'command': 'A', 'intermed': '', 'private': '', 'csi': '\x1b[', 'seq': '\x1b[2A'}
     True
     """
-    p = r"""(?P<front>.*?)
+    p = (
```

Comment:
Could you turn off black for this expression too, I can't stand what this does to the `[""" + "\x9b" + r"""]` bits which are really just a way to write \x9b in here.